src/std/rpc/ws_client/mod.rs

```diff
@@ -139,13 +139,27 @@ where
     }
 }
 
+const MAX_MESSAGE_LEN: usize = 1024;
+
+fn format_message(msg: &Message) -> String {
+    match (msg.as_text(), msg) {
+        (Ok(text), _) if text.len() <= MAX_MESSAGE_LEN => text.to_string(),
+        (Ok(text), _)  => format!("{}...", &text[..MAX_MESSAGE_LEN]),
+        (_, Message::Binary(bin)) if bin.len() <= MAX_MESSAGE_LEN => format!("Binary content: {:?}", bin),
+        (_, Message::Binary(bin)) => {
+            format!("Binary content: {:?}...", &bin[..MAX_MESSAGE_LEN])
+        }
+        _ => unreachable!("Only `Message::Binary` can fail `.as_text()`.")
+    }
+}
+
 pub fn on_get_request_msg(msg: Message, out: Sender, result: ThreadOut<String>) -> WsResult<()> {
     out.close(CloseCode::Normal)
         .unwrap_or_else(|_| warn!("Could not close Websocket normally"));
 
-    info!("Got get_request_msg {}", msg);
+    info!("Got get_request_msg {}", format_message(&msg));
     let result_str = serde_json::from_str(msg.as_text()?)
-        .map(|v: serde_json::Value| v["result"].to_string())
+        .map(|v: Value| v["result"].to_string())
```

Comment:
small nit: such changes can potentially make it harder to maintain fork

Reply:
reverted